plugins/notifications-backend/src/service/types.ts

```diff
@@ -30,9 +30,15 @@ export type Notification = {
   actions: NotificationAction[];
 };
 
+// Keep in sync with FE: plugins/notifications-frontend/src/api/notificationsApi.ts
 export type NotificationsFilter = {
   containsText?: string;
   createdAfter?: Date;
   messageScope?: 'all' | 'user' | 'system';
```

Comment:
I will replace this with string and do what you did with the allowed sort fields.